Neos.Flow/Classes/Session/Http/SessionRequestComponent.php

```diff
@@ -74,7 +74,8 @@ protected function prepareCookie(string $name, string $value)
     {
         return new Cookie(
             $name,
-            $value,
+            // @see https://github.com/neos/flow-development-collection/issues/2133
+            trim(urldecode($value), '"'),
```

Comment:
So we're okay with potentially removing too many double quotes? (see https://github.com/neos/flow-development-collection/issues/2133#issuecomment-694728094) Just trying to get your opinion on this. I know we already had it that way in `Headers`, but we could as well fix it there and here with this, no?

Reply:
In that part of the code `$value` contains the Flow session ID which consists of 32 basic characters (matching regex `/^[a-zA-Z0-9]{32}$/`). Plus, the enclosing quotes in some cases.
I think it's fine to fix just that case for now. If it turns out that other clients do other weird stuff we might have to decide differently

Reply:
Good point, didn't think about the specific use-case we have in the session component.

Reply:
That's actually also the case that threw the exception in our case since double quotes aren't a valid cache entry identifier so removing the double quotes entirely sounds ok to me.